### PR TITLE
Add option to write all output repos to file

### PR DESCRIPTION
**Pull request by rbikar**
New cli option intriduced --output-all-repos.
If provided along with --output-changed-repos, it will
write to output file all output repos.

### CODE REVIEW

File: tests/test_ubipop.py [hunk 1]
```diff
@@ -56,6 +69,18 @@ def get_test_mod(**kwargs):
                   kwargs.get('profiles', ''))
 
 
+def test_get_output_repo_ids(ubi_repo_set):
+    repo_ids = ubi_repo_set.get_output_repo_ids()
+    assert len(repo_ids) == 3
```
Comment (midnightercz) [inline tests/test_ubipop.py]: not sure if this makes sense when you test value directly one line bellow

File: tests/test_ubipop.py [hunk 2]
```diff
@@ -56,6 +69,18 @@ def get_test_mod(**kwargs):
...
+    assert repo_ids == set(["ubi-foo-rpms", "ubi-foo-source", "ubi-foo-debug"])
+
+
+def test_get_output_repo_ids_no_debug(ubi_repo_set_no_debug):
+    repo_ids = ubi_repo_set_no_debug.get_output_repo_ids()
+    assert len(repo_ids) == 2
```
Comment (midnightercz) [inline tests/test_ubipop.py]: same here

**Comment by rbikar** [comment]
@rohanpm 
Originally I thought we would like only changed repos in output file for pulp manifests. I agree, if there is no use case for that, it's not a good idea to add every possible option there. I'll leave only option for writing out all output repos.

**Comment by rbikar** [comment]
I decided to consolidate these output options into one which only makes sense.

**Pull request merged**